### PR TITLE
POl-352-Google-Schedule_instance_copy

### PR DESCRIPTION
**Pull request by KVivek03**
### Description

Modified the timezones_list.json file and remove foreach loop from the policy to reduce execution time.

### Issues Resolved

[List any existing issues this PR resolves]

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: cost/google/schedule_instance/google_schedule_instance.pt [hunk 1]
```diff
@@ -325,16 +325,7 @@ define schedule_instance($data) return $all_response do
...
-    foreach $obj in $timezones do
-      if $count==0
-        $var=to_a($obj)
-        if $var[0][0]==$timezone
-          $timezone=$var[0][1]
-          $count=$count+1
-        end
-      end
-    end
+	$timezone=$timezones[$timezone]
```
Comment (cdwilhelm) [inline cost/google/schedule_instance/google_schedule_instance.pt:328]: what happens if the $timezone isn't typed correctly or isn't found?

Comment (KVivek03) [reply cost/google/schedule_instance/google_schedule_instance.pt:328]: I have added the code to handle this. If the timezone is not found, sys log will show that error. Thanks

File: cost/google/schedule_instance/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v2.1
+
+- modified timezones_list.json file and optimized the code
```
Comment (douglaswth) [inline cost/google/schedule_instance/CHANGELOG.md:5]: As someone who might be actually using this Policy, this ChangeLog entry is very vague and does not really convey much. What was the purpose of the `timezones_list.json` change? Did it add new timezones? What was optimized in the code?

This is falling into the trap of "bug fixes and performance improvements" which is not very helpful.

Comment (KVivek03) [reply cost/google/schedule_instance/CHANGELOG.md]: updated the changelog with details

**Pull request merged**